File: Week-1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

/**
 * This program replicates the pyramid from the classic Mario game.
 * It prompts the user for the height of the pyramid, then prints a right-aligned pyramid of hashes (#) on the screen.
*/

int main(void)
{
    int num;

    do
    {
        num = get_int("Height: ");
    }
    while (num < 1);
    for (int i = 1; i <= num; i++)
    {
        for (int j = 0; j < num - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week-6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    x = get_int("Height: ")
    if x > 0 and x < 9:
        break

for i in range(1, x + 1):
    print(" " * (x - i), end="")
    for j in range(i):
        print("#", end="")
    print(" " * 2, end="")
    for k in range(i):
        print("#", end="")
    print()
